### ISSUE TITLE
Installer does not set memory for the wrapper in 3.0 RC1

### ISSUE THREAD

**Issue by lcahlander** [eXist-db/exist#765]
When installing eXist 3.0 RC1, I set the memory to 4096.  vm.properties has its values properly set, but wrapper/conf/wrapper.conf still has 1024 as the memory setting.

**Comment by dizzzz** [comment]
> When installing eXist 3.0 RC1
via the installer ?

> wrapper/conf/wrapper.conf still has 1024 as the memory setting.
Hmmm I'd not think of a direct relation between the installer and the wrapper config; You think you typically need to adjust this file anyway to get reasonably settings....

**Comment by dizzzz** [comment]
It has never been set by the installer, I updated the title

**Comment by ljo** [comment]
I am working on exposing this and choice of port(s) through the installer. It will take some time before I can get back to it though.

**Comment by adamretter** [comment]
@ljo Can we have an update please?

**Comment by ljo** [comment]
@adamretter Yes, I will check. But this is implemented unless someone broke it.

**Comment by adamretter** [comment]
@ljo Did you have a conclusion?

**Comment by line-o** [comment]
We are no longer maintaining YASW builds. I propose to close without further action.

**Comment by reinhapa** [comment]
> We are no longer maintaining YASW builds. I propose to close without further action.

@line-o should we not support this at least in the new service environment?

**Comment by dizzzz** [comment]
@reinhapa which settings would this update?

**Comment by line-o** [comment]
we decided to close this on the community call on 2nd March 2026
